enterprise-suite/Makefile

```diff
@@ -84,7 +84,15 @@ delete-local:  ## Delete chart from cluster with helm
 	-helm delete --purge $(RELEASE_NAME)
 
 install-local: $(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz install-helm delete-local  ## Install local chart
-	helm install $(HELM_CHARTS_DIR)/docs/$(RELEASE).tgz --name=$(RELEASE_NAME) --namespace=$(NAMESPACE) --debug --wait
+	@if test -z "$(BINTRAY_USERNAME)"; then \
```

Comment:
Since these are dev-related targets, do we want to try pulling them from some file if not explicitly specified?